data/category.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/influxdata/influx-stress/write"
 )
 
-func GetDataDirs(root, pattern string) []string {
+func GetDataDirs(root string) []string {
```

Comment:
There's a couple meta queries that are also a part of this. `show series`, `show field keys`, `show tag keys`, and all of the variants of those kinds of queries. 

I agree that data sounds wrong though. We could add the restriction that they be for selects and have a separate command for meta queries.